src/api-service/__app__/onefuzzlib/tasks/config.py

```diff
@@ -102,6 +104,12 @@ def check_target_exe(config: TaskConfig, definition: TaskDefinition) -> None:
 
         return
 
+    if (
```

Comment:
Maybe just add a comment reminding ourselves why we're doing this: we get relative paths as input, but blob storage doesn't have a notion of path. It deals in structureless string _names_, and so has no upstream notion of normalizing/traversal when comparing the equality of blob names.